src/components/Home/FeaturedBrands/FeatureBrands.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import "./FeaturedBrands.scss";
+import First from "../FeaturedBrands/BrandAssests/brand1.jpg";
+import Second from "../FeaturedBrands/BrandAssests/brand2.png";
+import Thrid from "../FeaturedBrands/BrandAssests/brand3.jpg";
+import Four from "../FeaturedBrands/BrandAssests/brand4.png";
```

Comment:
A common import here too can help. Have all the images inside one index and import from there

Reply:
I'm convert all images import statement into index.js file. thanks :)